### PR TITLE
fix(sdk): add generated file to vscode readonly config

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

**Review by fpaul-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json [hunk 1]
```diff
@@ -14,9 +14,13 @@
     "**/.yarn/**": true,
     <% } %>
     "**/node_modules/**": true,
-    "src/base/*/*.ts": true
+    "src/api/**/*.ts": true,
+    "src/base/**/*.ts": true,
+    "src/spec/api-mocks.ts": true,
```
Comment (matthieu-crouzet) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json:19]:
```suggestion
    "src/spec/api-mock.ts": true,
```

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json [hunk 2]
```diff
@@ -14,9 +14,13 @@
     "**/.yarn/**": true,
     <% } %>
     "**/node_modules/**": true,
-    "src/base/*/*.ts": true
+    "src/api/**/*.ts": true,
+    "src/base/**/*.ts": true,
```
Comment (matthieu-crouzet) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json:18]:
```suggestion
    "src/models/base/**/*.ts": true,
```

File: packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json [hunk 3]
```diff
@@ -14,9 +14,13 @@
...
+    "src/spec/api-mocks.ts": true,
+    "src/spec/operation-adapter.ts": true
   },
   "files.readonlyExclude": {
+    "src/api/*/index.ts": true,
     "src/base/*/index.ts": true
```
Comment (matthieu-crouzet) [inline packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/.vscode/settings.json:24]:
```suggestion
    "src/models/base/*/index.ts": true
```

**Comment by matthieu-crouzet** [comment]
Maybe we could apply it to our own repo 
and not only for the generated one :)

**Review by fpaul-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

**Pull request merged**